chromadb/__init__.py

```diff
@@ -255,3 +282,10 @@ def AdminClient(settings: Settings = Settings()) -> AdminAPI:
 
     """
     return AdminClientCreator(settings=settings)
+
+
+# Despite type hints, users may pass in non-string values for headers.
+def _stringify_headers(headers: Optional[Dict[str, str]]) -> Optional[Dict[str, str]]:
```

Comment:
I don't think we should be converting headers to strings here. Instead, we should let requests library error out on headers that are not strings () - https://github.com/psf/requests/issues/3491#issuecomment-238698353 and https://github.com/psf/requests/blob/72eccc8dd8b7c272e520f22b0256386c80864e94/src/requests/utils.py#L1040

Reply:
Sounds good, removed `_stringify_headers`